### ISSUE TITLE
Fix SEO: add metadata SEO keywords

### ISSUE THREAD

**Issue by fharper** [konstructio/kubefirst-docs#521]
### What type of suggestions?

Something's not working

### What page is concerned?

ALL

### What version of the docs?

Next

### What is the issue?

Add metadata SEO keywords in the front matter. It is in some pages, but missing in most.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by konstructcarrie** [comment]
Is this referring to this type of content for pages @fharper 

![image](https://github.com/user-attachments/assets/90901e88-918d-4632-97f4-5cb00f5eebee)

**Comment by fharper** [comment]
Yeah, the front matter (what is in between the `---`) at the top of mostly each pages (not in the partial ones).

They should all have the `description` and `keywords` filled properly to help with SEO as they will be rendered as meta HTML tags.
